doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -144,21 +144,21 @@ compatible with the rest of Coq, up to a few discrepancies:
   syntax of |SSR|’s rewrite and reserved identifiers are enabled
   only if the ssreflect module has been required and if ``SsrSyntax`` has
   been imported. Thus a file that requires (without importing) ``ssreflect``
-  and imports ``SsrSyntax``, can be required and imported without
+  and imports ``SsrSyntax`` can be required and imported without
   automatically enabling |SSR|’s extended rewrite syntax and
   reserved identifiers.
 + Some user notations (in particular, defining an infix ``;``) might
-  interfere with the "open term", parenthesis free, syntax of tactics
-  such as have, set and pose.
-+ The generalization of if statements to non-Boolean conditions is turned off
+  interfere with the "open term", parenthesis-free syntax of tactics
+  such as ``have``, ``set`` and ``pose``.
++ The generalization of ``if`` statements to non-Boolean conditions is turned off
   by |SSR|, because it is mostly subsumed by Coercion to ``bool`` of the
   ``sumXXX`` types (declared in ``ssrfun.v``) and the
   :n:`if @term is @pattern then @term else @term` construct
   (see :ref:`pattern_conditional_ssr`).  To use the
   generalized form, turn off the |SSR| Boolean ``if`` notation using the command:
   ``Close Scope boolean_if_scope``.
 + The following flags can be unset to make |SSR| more compatible with
-  parts of Coq:
+  parts of Coq.
```

Comment:
Again, I suggest a colon is better.

Reply:
I think, as commented in another PR, that colons shouldn't be followed by sentences, but only nominal groups.

Reply:
On the other hand, https://www.proofreadnow.com/blog/bid/91247/The-Colon-Its-Use-in-Punctuation#:~:text=With%20%22as%20follows%22%20and%20the,on%3B%20fourth%2C%20hold%20on! says

> The colon may be used instead of a period to introduce a series of related sentences (as in the fourth example below). 

> Sassafras was faced with a hideous choice: Should she reveal what was in the letter and ruin her reputation? Or should she remain silent and compromise the integrity of the judges?

Though in this case, it's not introducing sentences.

Reply:
Aren't they interrogative sentences?

Reply:
They are.  How does that matter?

Reply:
Oh, I see! I thought you were referring to the example, when you mentioned "this case". My mistake.